ImperatorToCK3/Source/CK3Outputter/CK3WorldOutputter.cpp

```diff
@@ -15,18 +19,44 @@ void createModFolder(const std::string& outputName);
 }
 
 
-void CK3::outputWorld(const World& CK3World)
+void CK3::outputWorld(const World& CK3World, const Configuration& theConfiguration)
 {
 	const auto& outputName = CK3World.getOutputModName();
 	createModFolder(outputName);
 	outputModFile(outputName);
 
-	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/history/");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/history");
 	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/history/provinces");
 	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/history/province_mapping");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common/landed_titles");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common/named_colors");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common/named_colors/imptock3");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common/coat_of_arms");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/common/coat_of_arms/coat_of_arms");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/localization");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/localization/replace");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/gfx");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/gfx/coat_of_arms");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/gfx/coat_of_arms/colored_emblems");
+	Utils::TryCreateFolder("output/" + CK3World.getOutputModName() + "/gfx/coat_of_arms/patterns");
 
 	LOG(LogLevel::Info) << "<- Writing Provinces";
 	outputHistoryProvinces(outputName, CK3World.getProvinces());
+
+	LOG(LogLevel::Info) << "<- Writing Landed Titles";
+	outputTitles(outputName, CK3World.getTitles());
+
+	LOG(LogLevel::Info) << "<- Writing Localization";
+	outputLocalization(outputName, CK3World);
+
+	LOG(LogLevel::Info) << "<- Copying named colors";
+	Utils::TryCopyFile(theConfiguration.getImperatorPath()+"/game/common/named_colors/default_colors.txt", "output/" + CK3World.getOutputModName() + "/common/named_colors/imp_colors.txt");
```

Comment:
what I did with game path was to integrate "/game/" into it so I don't have to type it manually. You could do the same for both imp and ck3.

Reply:
I chose the path to parent folder because that's consistent with older games, so it should be more intuitive for users who don't know the structure of the game files.

Reply:
Although I think we could append "/game" to the path right after reading the config.